libraries/dagster-delta/dagster_delta/io_manager.py

```diff
@@ -277,6 +281,7 @@ def connect(  # noqa: D102
             table_uri=table_uri,
             storage_options=storage_options or {},
             table_config=table_config,
+            parquet_read_options=parquet_read_options,
```

Comment:
We shouldn't save it in the TableConnection, it can be access from the resource_config from the context.

Reply:
Moved it out of the TableConnection